artview/components/display.py

```diff
@@ -0,0 +1,297 @@
+"""
+display.py
+
+Routines for Display modifications and additions.
+"""
+
+# Load the needed packages
+from ..core import QtGui, QtCore
+from .. import core
+from datetime import datetime as dt
+
+
+class ImageTextBox(core.Component):
+    '''
```

Comment:
This is not a real component, therefore it should not be a class instance of it. I believe a simple `QMainWindow` should be enough